### PR TITLE
Add weekly release issue automation

### PR DESCRIPTION
**Pull request by sophiewigmore**
## Summary

#642 

## Checklist

* [X] I have viewed, signed, and submitted the Contributor License Agreement.
* [X] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [ ] I have added an integration test, if necessary.
* [X] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [X] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

File: language-family/.github/workflows/create-release-issue.yml
Comment (robdimsdale) [inline language-family/.github/workflows/create-release-issue.yml:16]: it might be helpful to rename `month` to `day_of_month` or something similar.

Functionally this identical - and the current output is appropriate - but it would be easier to maintain if the variable name reflects the contents.

File: language-family/.github/workflows/create-release-issue.yml
Comment (robdimsdale) [inline language-family/.github/workflows/create-release-issue.yml:5]: This opens the issues on wednesday. Should we open the issues on monday instead? This would give maintainers the largest window possible to ship before end of week.

Also it means the issue will say: "Week of XX MM" where XX will be the date of a monday, which I think is slightly better than the date of wednesday of a given week.

**Review by robdimsdale — DISMISSED** [review]
❤️ huge thank you for implementing this!

**Comment by robdimsdale** [comment]
I have a couple of questions/thoughts (inline) but I'm happy to merge without making them if you prefer.

File: language-family/.github/workflows/create-release-issue.yml
Comment (sophiewigmore) [reply language-family/.github/workflows/create-release-issue.yml:16]: sure

File: language-family/.github/workflows/create-release-issue.yml
Comment (sophiewigmore) [reply language-family/.github/workflows/create-release-issue.yml:5]: I kind of like Wednesday. Coming in on Monday we usually have tons of issues to look at and organize. It's kind of nice to not have these come in until Wednesday

Comment (sophiewigmore) [reply language-family/.github/workflows/create-release-issue.yml:5]: I agree that the `Week of MM XX` being a Wednesday is slightly strange though

Comment (robdimsdale) [reply language-family/.github/workflows/create-release-issue.yml:5]: > My thought is that its kind of nice to not have these come in until Wednesday

Sure, I get that.

I don't agree - I'd prefer everything to come in Monday and let maintainers triage on their own pace - but I also don't feel strongly at all about this.

Regardless, I'd rather see this get merged and the automation start working.

**Review by robdimsdale — APPROVED** [review]

**Pull request merged**
